### PR TITLE
Add/Update README files

### PR DESCRIPTION
**Pull request by poulch**
## Checklist

- [ ] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: apps/cms-v2/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+4. CMS app will be available under `http://localhost:3000`
+
+> [!NOTE]
+> To test CMS Webhooks, you need to expose your local server to the internet (tunnel). You can use Saleor CLI to do that. See this [guide](https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-with-tunnels) for more details.
+
+5. Tunnel the app and install it on the Saleor dashboard.
+
+### Testing
+
+[Here](./docs/testing.md) you can find doc how to test the app
```
Comment (andrzejewsky) [inline apps/cms-v2/README.md:59]: I think this is not testing, this is rather configuration for each cms provider, I would recommend to rename it to something like "Providers configuration"

Comment (andrzejewsky) [reply apps/cms-v2/README.md:59]: it applies also for other apps's readme

File: apps/cms-v2/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+```shell
+pnpm dev
+```
+
+4. CMS app will be available under `http://localhost:3000`
+
+> [!NOTE]
+> To test CMS Webhooks, you need to expose your local server to the internet (tunnel). You can use Saleor CLI to do that. See this [guide](https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-with-tunnels) for more details.
+
+5. Tunnel the app and install it on the Saleor dashboard.
```
Comment (andrzejewsky) [inline apps/cms-v2/README.md:55]: can you attach the command for it?

Comment (andrzejewsky) [reply apps/cms-v2/README.md:55]: what does it mean "install it on the Saleor dashbaord", paste link to the manifest and install?

File: apps/klaviyo/README.md [hunk 3]
```diff
@@ -1,63 +1,57 @@
...
+<div style="text-align: center">
+  <img width="150" alt="" src="./public/logo.png">
+</div>
 
-Simple Klaviyo example app for Saleor
+<div style="text-align: center">
+  <h1>Saleor App Klaviyo</h1>
 
-![](docs/readme-assets/app-screen.png)
+  <p>Simple Klaviyo example app for Saleor</p>
```
Comment (andrzejewsky) [inline apps/klaviyo/README.md:8]: example app? we use it on prod so this is not "example" anymore

File: apps/cms-v2/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+#### Running app in development mode
+
+1. Install the dependencies by running the following command in the shell:
+
+```shell
+pnpm install
+```
+
+2. Create a file named `.env` and use the contents of the [`.env.example`](./.env.example) file as a reference. Set you `SECRET_KEY` in [`.env`](./.env)
```
Comment (andrzejewsky) [inline apps/cms-v2/README.md:42]: `.env.example` says that SECRET_KEY is required only for production build

Comment (andrzejewsky) [reply apps/cms-v2/README.md:42]: `APP_API_BASE_URL` and `APP_IFRAME_BASE_URL` - sounds like something to configure

File: apps/data-importer/docs/testing.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+## How to test Data importer app
```
Comment (andrzejewsky) [inline apps/data-importer/docs/testing.md:1]: this app is actually an integration with the nuvo, but you don't mention it at all

File: apps/data-importer/README.md [hunk 6]
```diff
@@ -1,5 +1,70 @@
...
+
+#### Running app in development mode
+
+1. Install the dependencies by running the following command in the shell:
+
+```shell
+pnpm install
+```
+
+2. Create a file named `.env` and use the contents of the [`.env.example`](./.env.example) file as a reference. Set you `NEXT_PUBLIC_NUVO_LICENSE_KEY` in [`.env`](./.env)
```
Comment (andrzejewsky) [inline apps/data-importer/README.md:29]: where can i that license key?

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

File: apps/smtp/docs/configuration.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+## How to configure the app:
+
+To configure app you need SMTP server, you can run it locally using docker
```
Comment (Cloud11PL) [inline apps/smtp/docs/configuration.md:3]:
```suggestion
To configure the app you need an SMTP server. You can run it locally using Docker.
```

**Review by Cloud11PL — APPROVED** [review]

File: apps/cms-v2/README.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+- [Strapi](https://strapi.io/)
+- [DatoCMS](https://www.datocms.com/)
+- [Builder.io](https://www.builder.io/)
+- [Payload](https://payloadcms.com/)
+
+### How to use this project
+
+#### Requirements
+
+- [node v16](http://nodejs.com)
```
Comment (krzysztofzuraw) [inline apps/cms-v2/README.md:28]:
```suggestion
- [node v20](http://nodejs.com)
```

**Review by krzysztofzuraw — COMMENTED** [review]
We can change node version to 20.x in READMEs

**Review by Cloud11PL — DISMISSED** [review]

**Review by krzysztofzuraw — DISMISSED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Review by Cloud11PL — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**